deeposlandia/datasets/aerial.py

```diff
@@ -0,0 +1,144 @@
+"""Aerial dataset module
+
+Model Aerial images for convolutional neural network applications. Data is
+downloadable at https://project.inria.fr/aerialimagelabeling/files/.
+
+"""
+
+from multiprocessing import Pool
+import numpy as np
+import os
```

Comment:
First the import of the stdlib, i.e. multiprocessing and os.

Then a new line and the other dependencies such as PIL and numpy.

Reply:
Fixed in all the modules.